ansible/roles/de-update-integration-data/tasks/main.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- name: update docker-py
+  pip: name="docker-py" state="latest"
+
+- name: pull the latest integrationator image
+  docker_image: name="{{ integrationator_repo }}" tag="{{ docker_tag }}" force=True
+
+- name: update integration data records
+  shell: >-
+    docker run --rm {{ integrationator_repo }}:{{ docker_tag }} -l {{ ldap_host }} -h {{ db_host }}
+    -p {{ db_port }} -d {{ db_name }} -U {{ db_user }} -P {{ db_password }}
```

Comment:
$ docker wait --help

    Usage:	docker wait [OPTIONS] CONTAINER [CONTAINER...]

    Block until a container stops, then print its exit code

      --help             Print usage

Possibly use this with [wait_for](https://docs.ansible.com/ansible/wait_for_module.html)?